system_tests/espresso_streamer_test.go

```diff
@@ -0,0 +1,97 @@
d4547444157674251695a517a575770303069664f44744a5653763141624f5363477244425342674e5648523845537a424a0a4d45656752614244686b466f64485277637a6f764c324e6c636e52705a6d6c6a5958526c63793530636e567a6447566b63325679646d6c6a5a584d75615735300a5a577775593239744c306c756447567355306459556d397664454e424c6d526c636a416442674e564851344546675155496d554d316c71644e496e7a673753560a55723951477a6b6e4271777744675944565230504151482f42415144416745474d42494741315564457745422f7751494d4159424166384341514577436759490a4b6f5a497a6a3045417749445351417752674968414f572f35516b522b533943695344634e6f6f774c7550524c735747662f59693747535839344267775477670a41694541344a306c72486f4d732b586f356f2f7358364f39515778485241765a55474f6452513763767152586171493d0a2d2d2d2d2d454e442043455254494649434154452d2d2d2d2d0a0000000000000b3a3d0000000000000163f90160f90159e10394a4b000000000000000000073657175656e636572837359818467934449c080b9013404f901308305172d8407270e008352d3f8948b14d287b4150ff22ac73df8be720e933f659abc80b8c43161b7f600000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000282e0000000000000000000000000000000000000000000000056bc75e2d63100001000000000000000000000000000000000000000000000000000000001b283e3e000000000000000000000000000000000000000000000000000000000000001084e4c2e4f5a0532b3903b248c357981dbdaddc8e22b137fd23f616ae153eee09ee80451ced37a047d5a4ac9ecd58fccbc8452bfa5e902733391239bdbb8871903c1c20c969c3518301caf300000000000b3a3e0000000000000163f90160f90159e10394a4b000000000000000000073657175656e636572837359818467934449c080b9013404f901308305172e8407270e008352d3f8948b14d287b4150ff22ac73df8be720e933f659abc80b8c43161b7f6000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000002833000000000000000000000000000000000000000000000000002e1a6545b13a8500000000000000000000000000000000000000000000000000000002e9a38636000000000000000000000000000000000000000000000000000000000000001084e4c2e4f5a0a5161a9b1e4299466c59713bcc5f27fe57e5bb1566255dfe17969df310b801cda06944c148f45ac00d07d3b36e72b5bfdb57bfbf85a1d219de984dda93a21aa7228301caf300000000000b3a3f0000000000000163f90160f90159e10394a4b000000000000000000073657175656e636572837359818467934449c080b9013404f901308305172f8407270e008352d3f8948b14d287b4150ff22ac73df8be720e933f659abc80b8c43161b7f6000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000002810000000000000000000000000000000000000000000000000002e13679ef3be6200000000000000000000000000000000000000000000000000000002e9a38636000000000000000000000000000000000000000000000000000000000000001084e4c2e4f6a0e1243adb737aab6847f0d1732b49c024ff02a406e02718077b9ce74eb010a32ca0577ad84a66a43280cfcff304a3effdc961755cb21192e53c255676f0370bf28d8301caf3"
```

Comment:
How this was generated? Looks confusing

Reply:
We got this from decaf explorer, this is from Rari